distribution/kernel/carbon-home/dbscripts/db2.sql

```diff
@@ -796,6 +796,17 @@ CREATE TABLE UM_HYBRID_USER_ROLE(
     PRIMARY KEY(UM_ID,UM_TENANT_ID)
 )/
 
+CREATE TABLE UM_HYBRID_GROUP_ROLE(
+    UM_ID DECIMAL(31,0) NOT NULL,
+    UM_GROUP_NAME VARCHAR(255) NOT NULL,
+    UM_ROLE_ID DECIMAL(31,0) NOT NULL,
+    UM_TENANT_ID DECIMAL(31,0) DEFAULT 0 NOT NULL,
+    UM_DOMAIN_ID DECIMAL(31,0) DEFAULT 0 NOT NULL,
+    UNIQUE(UM_GROUP_NAME,UM_ROLE_ID,UM_TENANT_ID),
+    FOREIGN KEY(UM_ROLE_ID,UM_TENANT_ID) REFERENCES UM_HYBRID_ROLE(UM_ID,UM_TENANT_ID) ON DELETE CASCADE,
+    PRIMARY KEY(UM_ID,UM_TENANT_ID)
+)/
+
```

Comment:
We will have to look at execution plans and add necessary indexes and all for optimization.
Please create a task to track such and attend

Reply:
Created a task https://github.com/wso2/product-is/issues/8595

Reply:
I think we need to write a trigger to implement auto-generated ID for UM_ID column similar to what we have done in UM_HYBRID_ROLE_TR1

Reply:
Fixed and updated the PR.